src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import { DateTime } from 'luxon';
+
+import BreadcrumbNav from 'components/BreadcrumbNav';
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+import PageWrapper from 'components/PageWrapper';
+import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import { AppState } from 'reducers/rootReducer';
+import { fetchSystemIntake } from 'types/routines';
+
+import './index.scss';
+
+const GovernanceReviewTeam = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { systemId } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchSystemIntake(systemId));
+  }, [dispatch, systemId]);
+
+  const systemIntake = useSelector(
+    (state: AppState) => state.systemIntake.systemIntake
+  );
+
+  const component = cmsDivisionsAndOffices.find(
+    c => c.name === systemIntake.requester.component
+  );
+
+  const requesterNameAndComponent = component
+    ? `${systemIntake.requester.name}, ${component.acronym}`
+    : systemIntake.requester.name;
+
+  return (
+    <PageWrapper className="easi-grt">
+      <Header />
+      <MainContent>
+        <section className="easi-grt__request-summary">
+          <div className="grid-container padding-y-2">
+            <BreadcrumbNav>
+              <li>
+                <Link to="/">Home</Link>
+                <i className="fa fa-angle-right margin-x-05" aria-hidden />
+              </li>
+              <li>{systemIntake.requestName}</li>
+            </BreadcrumbNav>
+            <dl className="easi-grt__request-info">
+              <div>
+                <dt>{t('intake:fields.requestName')}</dt>
+                <dd>{systemIntake.requestName}</dd>
```

Comment:
Do we need a default for if the user hasn't filled in a request name yet? For me, it just showed up as empty. Might be worth checking with design?

Reply:
I don't think so. If an intake is moved onto GRT review, it should have an intake name filled in.